.github/workflows/create_confluence_release_notes.yml

```diff
@@ -57,12 +69,23 @@ jobs:
           role-to-assume: ${{ secrets.TARGET_CLOUD_FORMATION_DEPLOY_ROLE }}
           role-session-name: github-actions
 
-      - shell: bash
-        name: get target tag
+      - name: Get target tag
         run: |
-          target_tag=$(aws cloudformation describe-stacks --stack-name ${{ inputs.TARGET_ENVIRONMENT }}-ci --query "Stacks[0].Tags[?Key=='version'].Value" --output text)
+          if [ -n "${{ inputs.TARGET_TAG }}" ]; then
+            target_tag="${{ inputs.TARGET_TAG }}"
+          else
+            target_tag=$(aws cloudformation describe-stacks --stack-name ${{ inputs.TARGET_ENVIRONMENT }}-ci --query "Stacks[0].Tags[?Key=='version'].Value" --output text)
+          fi
           export target_tag
           echo "target_tag=${target_tag}" >> "$GITHUB_ENV"
+                    
+      - name: Call to markJiraReleased lambda
```

Comment:
this needs to go after we have connected to dev account (line 90) as the release notes lambda is only in dev account